aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -173,7 +173,9 @@ In this section, you create a [Node.js](https://nodejs.org/) project to convert
    * The `output` property overrides the default value of `dist`. The bundle is instead emitted in the `wwwroot` directory.
    * The `resolve.extensions` array includes `.js` to import the SignalR client JavaScript.
 
-1. Copy the `src` directory from the [sample project](https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/tutorials/signalr-typescript-webpack/samples/8.x/SignalRWebpack/src/) into the project root. The `src` directory contains the following files:
+1. Create a new directory named `src` in the project root, `SignalRWebpack/`, for the client code.
+   
+1. Copy the `src` directory from the [sample project](https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/tutorials/signalr-typescript-webpack/samples/8.x/SignalRWebpack/src/) into the `src`. The `src` directory contains the following files:
```

Comment:
```suggestion
1. Copy the `src` directory and its contents from the [sample project](https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/tutorials/signalr-typescript-webpack/samples/8.x/SignalRWebpack/) into the project root. The `src` directory contains the following files:
```

Reply:
The issue here I think is that the link was to the content of the `src`, and should have been a level up, so that it was clear you are to copy the `src` directory to the root as instructed.  By linking to `src` itself, you could accidentally think you are to copy the contents of the directory only, rather than the directory `src` and its contents.  I clarified "and its contents" and fixed the link.  Thanks for bringing this up, it wasn't as clear as it could be.